src/OrbitGl/GlCanvas.cpp

```diff
@@ -176,11 +167,16 @@ void GlCanvas::MouseMoved(int x, int y, bool left, bool /*right*/, bool /*middle
   mouse_screen_y_ = mouse_y;
 
   // Pan
-  if (left && !im_gui_active_) {
-    world_top_left_x_ = world_click_x_ -
-                        static_cast<float>(mouse_x) / static_cast<float>(GetWidth()) * world_width_;
-    world_top_left_y_ = world_click_y_ + static_cast<float>(mouse_y) /
-                                             static_cast<float>(GetHeight()) * world_height_;
+  if (left && !picking_manager_.IsDragging()) {
```

Comment:
Why the dragging here? Were there a bug?

Reply:
`picking_manager_.IsDragging() == true` means that the user clicked on an element in the capture window and is dragging across it. In this case, there should be no panning happening in GlCanvas, the current element will handle any drag-related actions. This is the same logic as before, it was just moved from `CaptureWindow` to `GlCanvas`.